### DISCUSSION TITLE
Create release ignores build mode Translated?

### DISCUSSION THREAD

**Discussion by cegekaJG** [microsoft/AL-Go#900] [Q&A] [answered]
I recently switched the build mode of a repo from default (previously no `buildModes` given specified) to "Translated".
The AL pipeline workflow seemed to work just fine, but when I tried to create a new release earlier today, I noticed that AL Go excludes the artifacts of the build mode "Translated" and tries to publish the apps from the last default build.
What exactly is the difference between the two build modes? I've deployed apps compiled with the default mode before, and I'm pretty sure they also include translations. But if the release workflow expects artifacts from the default build, what's the point of the other ones?

**Comment by freddydk** [comment, marked as answer]
Build Modes have been modified quite a lot. You can now create multiple build modes and setup preProcessor symbols per build mode.
Every buildmode will also have their own set of artifacts published, with different names.
